src/collective/listmonk/testing.py

```diff
@@ -23,6 +24,32 @@ def setUp(self):
             close_fds=True,
         )
 
+        # Configure SMTP server; disable unsubscribe headers
+        settings = listmonk.call_listmonk("get", "/settings")["data"]
+        smtp = settings["smtp"][0]
+        smtp.update(
+            {
+                "enabled": True,
+                "host": "mailhog",
+                "port": 1025,
+                "auth_protocol": "none",
+                "tls_type": "none",
+            }
+        )
+        settings.update({"smtp": [smtp], "privacy.unsubscribe_header": False})
+        listmonk.call_listmonk("put", "/settings", json=settings)
+
+        # Configure template
+        listmonk.call_listmonk(
+            "put",
+            "/templates/1",
+            json={
+                "name": "Default campaign template",
+                "type": "campaign",
+                "body": '{{ template "content" . }}',
+            },
+        )
+
     def tearDown(self):
         subprocess.call(
```

Comment:
Same thing here. Take a look at kitconcept.solr, we have a `pytest-docker` dependency that provides fixtures to handle Docker compose

Reply:
I started to try it, but it requires more code to wait for the services to come up, since it does not call `docker compose up` with the `--wait` flag. We can come back to this later.